src/plugins/intel_gpu/src/graph/eltwise.cpp

```diff
@@ -469,4 +469,37 @@ void eltwise_inst::check_inputs_count(eltwise_node const& node) {
             break;
     }
 }
+
+bool eltwise_node::need_input_tensors_size_align() const {
+    if (get_input_layouts().size() < 2)
+        return false;
+
+    auto pshape_a = get_input_pshape(0).size();
+    auto pshape_b = get_input_pshape(1).size();
+    if (pshape_a != pshape_b && pshape_a > 1 && pshape_b > 1 &&
+        get_primitive()->broadcast_spec == ov::op::AutoBroadcastType::NUMPY) {
+        return true;
+    }
+
+    return false;
+}
+
+bool eltwise_node::has_eltwise_const_dep_idx() const {
```

Comment:
How about combining the two functions into one? It doesn't look good to have them split into two. How about using std::optional?
```
std::optional<size_t> eltwise_node::find_first_const_dependency_index() const {
    for (size_t i = 0; i < get_dependencies().size(); ++i) {
        if (get_dependency(i).is_constant()) {
            return i;
        }
    }
    return std::nullopt;
}
```